modules/bitgo/test/v2/unit/internal/tssUtils/bitgoMpcGpgPubKeys.ts

```diff
@@ -0,0 +1,169 @@
+import {
+  BitgoMpcGpgPubKeys,
+  common,
+  ECDSAUtils,
+  EddsaUtils,
+  EnvironmentName,
+  IRequestTracer,
+  Wallet,
+} from '@bitgo/sdk-core';
+import { TestBitGo } from '@bitgo/sdk-test';
+
+import { BitGo } from '../../../../../src';
+import * as openpgp from 'openpgp';
+import nock = require('nock');
+import assert = require('assert');
+
+class TestEcdsaMpcv2Utils extends ECDSAUtils.EcdsaMPCv2Utils {
+  public async testPickBitgoPubGpgKeyForSigning(
+    isMpcv2: boolean,
+    reqId?: IRequestTracer,
+    enterpriseId?: string
+  ): Promise<openpgp.Key> {
+    return this.pickBitgoPubGpgKeyForSigning(isMpcv2, reqId, enterpriseId);
+  }
+}
+
+class TestEddsaMpcv1Utils extends EddsaUtils {
+  public async testPickBitgoPubGpgKeyForSigning(
+    isMpcv2: boolean,
+    reqId?: IRequestTracer,
+    enterpriseId?: string
+  ): Promise<openpgp.Key> {
+    return this.pickBitgoPubGpgKeyForSigning(isMpcv2, reqId, enterpriseId);
+  }
+}
+
+describe('TSS MPC Pick BitGo GPG Pub Key Utils:', async function () {
```

Comment:
`describe async` means everthing inside is a noop

I'm very surprised this isn't an eslint rule

Reply:
you're right did some reading and mocha doesnt handle a promise returned by describute

Reply:
great catch

Reply:
fixed